src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/01 Executors/06 ramping-arrival-rate.md

```diff
@@ -11,6 +11,14 @@ k6 will attempt to dynamically change the number of VUs to achieve the configure
 
 See the [arrival rate](/using-k6/scenarios/arrival-rate) section for details.
 
+<Blockquote mod="Note" title="Iteration starts are spaced fractionally">
+
+Iterations **do not** started at exactly the same time.
+At a `rate` of 10 with a `timeUnit` of `1s`, each iteration starts about every tenth of a second (that is, each 100ms).
+This jitter accidentally concurrency where all VUs start at the exact millisecond
```

Comment:
Something is wrong with this sentence, but I am not sure exactly what :sweat_smile:

Reply:
Oh I know what's wrong: it's not a sentence. It has no main verb. It should be "This jitter prevents." Fixing now...